src/Types/AbstractTypedIDEntity.php

```diff
@@ -4,31 +4,30 @@
 
 namespace AvtoDev\IDEntity\Types;
 
+use function mb_strlen;
+use function mb_substr;
```

Comment:
Maybe just from the `\`?
`\mb_strlen`
`\mb_substr`

Reply:
There are a lot of this functions calling in a class. So, `use function` is a better choice for clean code, I think